src/main/resources/hudson/plugins/plot/PlotReport/index.jelly

```diff
@@ -54,7 +54,7 @@
         <div style="width:750px">
           <j:if test="${it.getPlotDescription(index) != null}">
             <j:if test="${!it.getPlotDescription(index).isEmpty()}">
-              <b>Description</b>: <j:out value="${it.getPlotDescription(index)}"/>
+              <b>Description</b>: <span>${it.getPlotDescription(index)}</span>
```

Comment:
Doesn't even need the `span`. 

```
              <b>Description</b>: ${it.getPlotDescription(index)}
```

would do it.

Note that `j:out` specifically disables the effect of `escape-by-default` and prints it verbatim.